scripts/correct_error_ids.py

```diff
@@ -117,6 +117,7 @@ def find_source_files(top_dir):
 
 
 def main():
+    random.seed(datetime.now())
```

Comment:
Could we use just `random.seed()`?

Reply:
Yes. Didn't know that systime is used as default! I'll fix this.